_vector-search/tutorials/semantic-search/semantic-search-cfn-bedrock.md

```diff
@@ -0,0 +1,155 @@
+
+Replace the placeholders beginning with the prefix `your_` with your own values.
+{: .note}
+
+## Prerequisite: Create an OpenSearch cluster
+
+Go to the [Amazon OpenSearch Service console](https://console.aws.amazon.com/aos/home) and create an OpenSearch domain.
+
+Note the domain Amazon Resource Name (ARN); you'll use it in the following steps.
+
+## Step 1: Map a backend role
+
+The OpenSearch CloudFormation template uses a Lambda function to create an AI connector with an AWS Identity and Access Management (IAM) role. You must map the IAM role to `ml_full_access` to grant the required permissions. Follow [Step 2.2 of the Semantic search using Amazon Bedrock Titan tutorial]({{site.url}}{{site.baseurl}}/ml-commons-plugin/tutorials/semantic-search/semantic-search-bedrock-titan/#step-22-map-a-backend-role) to map a backend role.
+
+The IAM role is specified in the **Lambda Invoke OpenSearch ML Commons Role Name** field in the CloudFormation template. The default IAM role is `LambdaInvokeOpenSearchMLCommonsRole`, so you must map the `arn:aws:iam::your_aws_account_id:role/LambdaInvokeOpenSearchMLCommonsRole` backend role to `ml_full_access`.
+
+For a broader mapping, you can grant all roles `ml_full_access` using a wildcard:  
+
+```
+arn:aws:iam::your_aws_account_id:role/*
+```  
+
+Because `all_access` includes more permissions than `ml_full_access`, mapping the backend role to `all_access` is also acceptable.
+
+## Step 2: Run the CloudFormation template  
+
+The CloudFormation template integration is available in the [Amazon OpenSearch Service console](https://console.aws.amazon.com/aos/home). From the left navigation pane, select **Integrations**, as shown in the following image.
+
+![Semantic search CloudFormation integration]({{site.url}}{{site.baseurl}}/images/vector-search-tutorials/semantic_search_bedrock_integration_1.png)  
+
+To create a connector, complete the following form.
+
+![Deploy a pretrained model to Amazon Bedrock]({{site.url}}{{site.baseurl}}/images/vector-search-tutorials/semantic_search_bedrock_integration_2.png)
+
+Complete the following fields, keeping all other fields at their default values:  
+
+1. Enter your **Amazon OpenSearch Endpoint**.  
+2. In the Model configuration, select a **Model** to be deployed. Choose one of the following supported models: 
+    - `amazon.titan-embed-text-v1`
+    - `amazon.titan-embed-image-v1`
+    - `amazon.titan-embed-text-v2:0` 
+    - `cohere.embed-english-v3`
+    - `cohere.embed-multilingual-v3`
+3. Select a **Model Region** (this is the Amazon Bedrock service region).
```

Comment:
If "service region" refers to an AWS Region, please change to "Amazon Bedrock Region".